platform-dom/src/main/resources/com/softicar/platform/dom/dom-image-viewer-style.css

```diff
@@ -1,16 +1,35 @@
 .DomImageViewer {
 	align-items: center;
+	display: grid;
+	grid-template-rows: min-content auto;
+	overflow: hidden;
+	resize: both;
```

Comment:
By also adding `min-width: min-content;` to this class, we could avoid that the toolbar gets cropped.
![image](https://user-images.githubusercontent.com/15086658/188497348-09c2fba4-3b57-4862-b444-07952aa2ef35.png)

Reply:
good idea, did so